daprdocs/content/en/developing-applications/develop-components/pluggable-components/develop-pluggable.md

```diff
@@ -105,6 +105,167 @@ After generating the above state store example's service scaffolding code using
 
 This concrete implementation and auxiliary code are the **core** of your pluggable component. They define how your component behaves when handling gRPC requests from Dapr.
 
+## Return semantic state store errors
```

Comment:
Although we don't have any other errors than those returned by state store components, I think we should name this section more broadly e.g `Returning semantic errors`.

And then, we can add one more column in the table below specifying the source component, how about that?

Reply:
Sounds good to me